### DISCUSSION TITLE
Add an configuration parameter about how Nemo handle when moving a file with the mouse and releasing just on an iso or zip file

### DISCUSSION THREAD

**Discussion by osgpcq** [linuxmint/discussions#221] [Ideas]
Hi,
Not 100% if I'm in the correct place.
So with Cinnamon using Nemo 5.8.5, when I want to move some files for example from one tab to another one, if in the destination directory, some iso or zip files are present, and I release the file on a compressed or iso file, the file is not moved as expected but added in the iso or zip!

So I'm afraid to have some files disappearing (well hidden in some compressed or iso files),
and my zip or iso files with some files not wanted at all into.

At this point, I didn't find an option to change the behavior...
Any ideas where I can change this?
If nothing is already present, do you think a parameter could be added for this? (in my point of view it's really bad to be sure to keep files as expected).

**Comment by mtwebster** [comment]
What you're describing is actually considered a 'feature' (dropping files onto compressed files tries to add the file) - but your issue is related to a more general problem where you can't drop a file into the current location if view is filled with folders (or zip files).

We merged some changes a week or so ago that will now cause drops into any empty part of the background, regardless of row, to be considered a drop into the current location.

see:
https://github.com/linuxmint/nemo/commit/b9d396e153951b1925e99105857f47d3261a06cb

This is part of the upcoming 6.0 release. The compressed file behavior will remain, but it will require more deliberate action.

**Comment by osgpcq** [comment]
You're right I suppose if I drop the file in the background and not directly onto the file, this will solve my issue and I will be able to use the feature as well :)
Thank you!
